src/ensembl/src/content/app/about-ensembl/AboutEnsembl.scss

```diff
@@ -0,0 +1,7 @@
+@import 'src/styles/common';
+
+.aboutEnsembl {
+  height: 100%;
+  display: grid;
+  grid-template-rows: auto 1fr;
```

Comment:
Why is it a grid with rows?

![image](https://user-images.githubusercontent.com/6834224/76009135-9efd5100-5f08-11ea-907e-5e0e446de33a.png)

Reply:
Yeah, I probably need to take that out - I just copied everything from StandardAppLayout!